torch/csrc/jit/codegen/cuda/test/test_gpu3.cpp

```diff
@@ -6507,6 +6507,39 @@ TEST_F(NVFuserTest, FusionSimpleAmperePipeline_CUDA) {
   testValidate(&fusion, cg_outputs, {input1}, {input1}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, FusionVectorizeRepro_CUDA) {
+  std::unique_ptr<Fusion> fusion_ptr = std::make_unique<Fusion>();
+  auto fusion = fusion_ptr.get();
+  FusionGuard fg(fusion);
+
+  TensorView* tv1 =
+      TensorViewBuilder().ndims(2).contiguity({true, true}).build();
+  TensorView* tv0 =
+      TensorViewBuilder().ndims(2).contiguity({true, true}).build();
+  fusion->addInput(tv1);
+  fusion->addInput(tv0);
+
+  auto tv7 = sum(tv0, {1}, true);
+  auto tv_exp =
+      expand(tv7, {tv0->axis(0)->extent(), IrBuilder::create<Int>(32128)});
+  auto tv3 = exp(tv1);
+  auto tv8 = mul(tv3, tv_exp);
+  auto tv13 = sub(tv0, tv8);
+  fusion->addOutput(tv13);
```

Comment:
Does this actually reproduce the issue? It seems this PR addresses an issue with validation vectorization of input and output tensors with an expanded broadcast IterDomain. This test has an expanded broadcast, but that isn't in any of input or output tensors. Am I missing anything?

Reply:
Haha, the test is tricky, there's a fusion segmentation happening right at the expand op.

I tried to get a repro without that segmentation, somehow that doesn't trigger the issue cold_sweat

Reply:
Hmm. Can you please add that as a comment to the test? It'd be great if we could understand why it wouldn't fail with segmentation.